packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx

```diff
@@ -60,7 +60,7 @@ const StripePaymentLegalBlurb = () => (
 );
 
 const DefaultPaymentLegalBlurb = () => (
-  <div className="payment-legal-blurb">
+  <div className="payment-legal-blurb clear-both font-medium text-center">
```

Comment:
/summons @LZoog
Since we're reusing these same 3 styles for each instance of `.payment-legal-blurb`, can we do some sort of Tailwind-ism here like adding `@apply clear-both font-medium text-center;` to `.payment-legal-blurb` directly? Or doesn't it work like that?

Reply:
This was updated and now includes the same styles as used in TermsAndPrivacy. It was used once there, but has now been repeated 3 times in this file. I agree that it would be good to add to `/packages/fxa-payments-server/src/styles/tailwind.css`. It's a small lift but shall I add to this ticket, or file a new one @LZoog @StaberindeZA @xlisachan ?

Reply:
I'm okay with it being added to this PR and just revising the ticket to call out the additional update. I feel like we could potentially use this in emails one day. :)

Reply:
Updated and added to this PR :) .